natives.json

```diff
@@ -37812,15 +37812,15 @@
 			"build": "1207"
 		},
 		"0xE9F676788F8D5E1E": {
-			"name": "_0xE9F676788F8D5E1E",
+			"name": "_IS_BLIP_ON_MAP",
```

Comment:
This does not check whether the blip is on the map. It actually checks if the blip is attached to any entity.
So ``_IS_BLIP_ATTACHED_TO_ANY_ENTITY`` would be better.